frontend/lib/verification/verification_view.dart

```diff
@@ -0,0 +1,226 @@
+import 'dart:ui';
+
+import 'package:flutter/cupertino.dart';
```

Comment:
```suggestion
```

Just a trifle: we should not import `cupertino` anywhere. I know, IntelliJ likes to import it very much. Probably also in the other UI related files.

Reply:
I never ever imported that thing manually :D